plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -46,4 +46,44 @@
     </listener>
   </extension>
 
+  <extension point="org.eclipse.ui.commands">
+    <command
+      description="%convertToAppEngineStandardCommandDescription"
+      id="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard"
+      name="%convertToAppEngineStandardCommandName">
+    </command>
+  </extension>
+
+  <extension point="org.eclipse.ui.menus">
+    <menuContribution
+      allPopups="false"
+      locationURI="popup:org.eclipse.ui.popup.any?after=additions">
+      <command
+        commandId="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard"
+        label="%convertToAppEngineStandardMenuLabel">
+        <visibleWhen checkEnabled="true" />
+      </command>
+    </menuContribution>
+  </extension>
+
+  <extension point="org.eclipse.ui.handlers">
+    <handler
+      class="com.google.cloud.tools.eclipse.appengine.facets.ui.AppEngineStandardProjectConverter"
+      commandId="com.google.cloud.tools.eclipse.appengine.facets.convertToStandard">
+      <enabledWhen>
+        <and>
+          <count value="1" />  <!-- enabled when only one project is selected -->
+          <iterate>
+            <adapt type="org.eclipse.core.resources.IProject" />
+          </iterate>
+          <not>
+            <iterate>
+              <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
If the project doesn't have a WEB-INF then we add one, right? along with web.xml and appengine-web.xml?